models/registry/api.ts

```diff
@@ -173,10 +174,14 @@ export async function getUnchartedRealmInfos(connection: ConnectionContext) {
   ).map((p) => {
     return p.programId
   })
-
+  const programIdsChunk = chunks(programIds, 5)
```

Comment:
note: `chunks` from `@utils/helpers`  and `group` and `@utils/group` seems to do the same thing.

Reply:
Hi actually chunk is not helping so we will use indexer waiting for them to update query for us :)